docs/visualize.asciidoc

```diff
@@ -1,2 +1,131 @@
 [[visualize]]
-== Visualizing your Data
\ No newline at end of file
+== Visualize
+
+You can use Kibana's _Visualize_ app to design and create data visualizations. You can save these visualizations, use
+them individually, or combine visualizations into a _dashboard_. A visualization can be based on one of the following 
+data source types:
+
+* A new interactive search
+* A saved search
+* An existing saved visualization
+
+Visualizations are based on the {ref}/search-aggregations.html[aggregation] feature introduced in Elasticsearch 1. 
```

Comment:
You just said they were based on a search or visualization. Maybe "are constructed using"?